docs/app/pages/components/components-sections/input-controls/radio-button/radio-button.component.html

```diff
@@ -1,7 +1,6 @@
-<div role="radiogroup" class="radio-button-container">
+<div role="radiogroup" uxRadioButtonGroup [(value)]="selected" class="radio-button-container">
```

Comment:
`uxRadioButtonGroup` should probably be added to the usage popover

Reply:
added to popover